src/components/FilterDropdown.vue

```diff
@@ -408,6 +425,22 @@ export default {
             emit('input', [])
         }
 
+        function loadMore() {
+            const toSet = state.itemsVisible + props.itemsPerPage
+            state.itemsVisible = toSet > parsedItems.value.length ? parsedItems.value.length : toSet
+            // Scroll the list items container to the bottom when items are added
+            root.$nextTick(() => {
+                uiGridEl.value.scrollTop = uiGridEl.value.scrollHeight
+            })
+        }
+
+        const showLoadMore = computed(() => {
+            if (props.itemsPerPage && parsedItems.value.length !== state.itemsVisible) {
+                return true
```

Comment:
Here we could directly return ```props.itemsPerPage && parsedItems.value.length !== state.itemsVisible``` probably, just as an option. Looking really good otherwise.